pkg/synchromanager/clustersynchro_manager.go

```diff
@@ -604,12 +604,12 @@ func buildClusterConfig(cluster *clusterv1alpha2.PediaCluster) (*rest.Config, er
 	}
 
 	if cluster.Spec.APIServer == "" {
-		return nil, errors.New("Cluster APIServer Endpoint is required")
+		return nil, errors.New("cluster APIServer Endpoint is required")
```

Comment:
Here they are treated as a special whole that can begin with a capitalization.

Reply:
ok, I got it, thank you for your review.

Reply:
@Iceber  Already modified, thanks for reviewing again.